sql: unhandled node type commentOnDatabaseNode panic

Found via the high vmodule sql logic test nightly.

```
panic: unhandled node type: *sql.commentOnDatabaseNode [recovered]
	panic: panic while executing 1 statements: COMMENT ON DATABASE _ IS 'foo'; caused by unhandled node type: *sql.commentOnDatabaseNode

goroutine 47427 [running]:
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).closeWrapper(0xc003a64000, 0x3212000, 0xc0046249c0, 0x2777b20, 0xc001706750)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:721 +0x36d
github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn.func1(0xc003a64000, 0x3212000, 0xc0046249c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:434 +0x61
panic(0x2777b20, 0xc001706750)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
github.com/cockroachdb/cockroach/pkg/sql.planPhysicalProps(0x3212f80, 0xc001706680, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/plan_physical_props.go:146 +0x14a5
github.com/cockroachdb/cockroach/pkg/sql.planToString(0x32120c0, 0xc003aae3f0, 0x3212f80, 0xc001706680, 0x0, 0x0, 0x0, 0x0, 0x3212f80)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/explain_plan.go:378 +0x338
github.com/cockroachdb/cockroach/pkg/sql.(*planner).makePlan(0xc003a64350, 0x32120c0, 0xc003aae3f0, 0x2b73087, 0x12)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/plan.go:378 +0x59d
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).makeExecPlan(0xc003a64000, 0x32120c0, 0xc003aae3f0, 0xc003a64350, 0x13, 0x89ed5f)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:786 +0x302
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc003a64000, 0x32120c0, 0xc003aae3f0, 0xc003a64350, 0x7fd3047526f0, 0xc000771080, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:648 +0x148
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc003a64000, 0x32120c0, 0xc003aae3f0, 0x32160c0, 0xc0042138c0, 0xc0012dbae0, 0x21, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:412 +0xafe
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc003a64000, 0x32120c0, 0xc003aae3f0, 0x32160c0, 0xc0042138c0, 0xc0012dbae0, 0x21, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:100 +0x610
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc003a64000, 0x32120c0, 0xc003aae390, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1200 +0x1ad8
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc003a64000, 0x3212000, 0xc0046249c0, 0xc000a1e338, 0x5400, 0x15000, 0xc000a1e3d0, 0xc000a71ff0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1136 +0x1b4
github.com/cockroachdb/cockroach/pkg/sql.(*Server).ServeConn(0xc000107b80, 0x3212000, 0xc0046249c0, 0xc003a64000, 0x5400, 0x15000, 0xc000a1e3d0, 0xc000a71ff0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:436 +0xce
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync.func1(0xc001281b01, 0xc0024198a0, 0x3212000, 0xc0046249c0, 0xc000a71ff0, 0xc000107b80, 0xc000dbe000, 0x3215540, 0xc002419880, 0xc002d97200, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:580 +0x21f
created by github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).processCommandsAsync
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:515 +0x17b
```

--- a/pkg/sql/plan_physical_props.go
+++ b/pkg/sql/plan_physical_props.go
@@ -103,6 +103,7 @@ func planPhysicalProps(plan planNode) physicalProps {
 	case *cancelSessionsNode:
 	case *commentOnTableNode:
 	case *commentOnColumnNode:
+	case *commentOnDatabaseNode:
 	case *controlJobsNode:
 	case *createDatabaseNode:
 	case *createIndexNode:

